metaflow/metaflow_config.py

```diff
@@ -312,6 +312,9 @@
 ARGO_EVENTS_INTERNAL_WEBHOOK_URL = from_conf(
     "ARGO_EVENTS_INTERNAL_WEBHOOK_URL", ARGO_EVENTS_WEBHOOK_URL
 )
+ARGO_EVENTS_SENSOR_NAMESPACE = from_conf(
+    "ARGO_EVENTS_SENSOR_NAMESPACE", KUBERNETES_NAMESPACE
```

Comment:
Isn't it confusing to put KUBERNETES_NAMESPACE value here?

Reply:
This basically ensures that the default behavior is the current behavior.  You wouldn't want to set the `KUBERNETES_NAMESPACE` but then have this be some sort of default namespace.  Why do you think it is confusing?